### ISSUE TITLE
Loss Function and Reproducing ImageNet Classification Results

### ISSUE THREAD

**Issue by zcablii** [qhfan/RMT#9]
I have a question regarding the loss function used in the classification task described in your paper. In the paper, it is mentioned that "the only supervision is classification loss for a fair comparison." However, upon examining the provided codebase, it seems that the default configuration and script are using 'regnety_160' as a teacher model. I am curious to know whether the reported results in the paper were obtained using the teacher model or if they were based solely on a simple classification CrossEntropy loss.

If I aim to reproduce the ImageNet classification results, can I simply replace the distillation loss with a CrossEntropy loss? Are there any other differences or considerations? Thank you!

**Comment by qhfan** [comment]
Thanks for your attention to our work. The args.distillation_type in our code is set to "None". The distillation loss showed in our code is just a wrapper. In fact, the code we released only use the naive cross entropy loss. You can directly use the code without any modification.
